exercises/concept/boutique-inventory/.docs/instructions.md

```diff
@@ -15,9 +15,9 @@ A single item in the inventory is represented by a hash, and the whole inventory
 ]
 ```
 
-## 1. Return a list of the names of the items in stock
+## 1. Return a list of the names of the items.
```

Comment:
I was right about being unsure about the period.  This is the only heading with a period.  I think the period does not fit the style for this document in the heading.

Reply:
yeah, the linter in my IDE flagged it and I overlooked it.  good catch!